packages/staking-contracts/src/rewards-allocator.ts

```diff
@@ -38,11 +38,13 @@ class RewardsAllocator {
   private walletClient: WalletClient;
   private publicClient: PublicClient;
   private contractAddress: Hex;
+  private tokenAddress: Hex;
 
   constructor(
```

Comment:
minor change in this class to accept `tokenAddress` in the constructor and not in `allocate` method